src/AffectedUserFlow/CodeInput/CodeInputForm.tsx

```diff
@@ -116,61 +124,86 @@ const CodeInputForm: FunctionComponent = () => {
 
   const isDisabled = code.length !== codeLength
 
+  const codeInputBorderStyle = isFocused
+    ? { borderColor: Colors.secondaryViolet }
+    : null
+  const codeInputStyle = { ...style.codeInput, ...codeInputBorderStyle }
```

Comment:
This is minor, but it feels a little difficult to follow having style declarations up here and also down below in the `style` definition. What do you think about refactoring `styles` into a function that can take an argument like `codeInputIsFocused` and return the correct styles? Then up here you can just call `const styles = getStyles(isFocused)`, and have all of the styles being set in one place?

Reply:
@achen178 and I paired on an improvement to this, but I had trouble understanding exactly what you're recommending here. I'm going to merge this in for now, but would love to pair on it later perhaps!